workflow-templates/im-deploy-on-prem-database.yml

```diff
@@ -170,28 +171,60 @@ jobs:
         env:
           DB_NAME@dev: ''
           DB_NAME@qa: ''
-          DB_NAME@stage demo: ''
+          DB_NAME@stage demo uat: ''
           DB_NAME@prod: ''
           DB_SERVER_NAME@dev: ''
           DB_SERVER_NAME@qa: ''
-          DB_SERVER_NAME@stage demo: ''
+          DB_SERVER_NAME@stage demo uat: ''
           DB_SERVER_NAME@prod: ''
+          VAULT_URL@dev qa: 'https://vault.dev.mktp.io:8200'
+          VAULT_URL@stage demo uat prod: 'https://vault.mktp.io:8200/'
```

Comment:
What do you think about TODOs for both of these lines indicating the user needs to verify the vault url for their environments since it's unclear how people are using the two vault environments.

Reply:
That seems like a good idea. I was thinking the general TODO at the top of the step that says to fill out the values would be good enough, but since these look filled out already I think it would be good to add something specifically for them.